### ISSUE TITLE
[RFC] SyliusVariableProductBundle Options with no impact on price

### ISSUE THREAD

**Issue by thomas2411** [Sylius/Sylius#1020]
Hi,

I am developing a shop which uses many parameters to define a product. There is a photo album product which is defined with few parameters. Some of them should have impact on price, some of them, not. 
## Options with no impact on variants and price

For example there are: 

| Size | Cover | Price |
| --- | --- | --- |
| 20x20 | Photo | 50.00 |
| 20x20 | Classic | 45.00 |
| 20x30 | Photo | 60.00 |
| 20x30 | Classic | 55.00 |

When I Generate variants based on these settings, I get 4 variants. If I use 8 sizes and 4 _Cover types_...well, I get a lot of variants to manage.
Now, the clients needs to add some parameters which don't have impact on price, like _Cover inner color_ This parameter should be as a Select option added to each variant. But it shouldn't generate a separate variant for each of _Cover inner color_. I need to develop such functionality but I would like to add it later to **SyliusVariableProductBundle** if authors agree to add it. 

My question is: should I change Options/Variants somehow to add a boolean which will define if this particular Option should be taken into account while generating variants and should have price or not? Or should I generate another item. Instead of two: Options and Variants, there would be three: Options, Variants and for instance Type? 
## Fixed option

Another thing is for instance engraver for album. This is an option which has fixed price for all products. So when user checks a checkbox, then engraver is added to album product in order and prices is changed. It's not making sense to add this as a normal option because it will generate one more variant for each of other options. SO as in previous question, how should I add it.
### To sum up

Both things are very similar: options with no impact on price which behaves differently from normal options and options with fixed prices for all selected products. Should I:
1. change Options and Variants?
2. add another part for this Bundle?
3. add another Bundle which extends SyliusVariableProductBundle
4. another solution?
